formwidgets/MLNestedForm.php

```diff
@@ -21,13 +21,37 @@ class MLNestedForm extends NestedForm
      */
     protected $defaultAlias = 'mlnestedform';
 
+    /**
+     * The nestedform translation mode (default|fields)
+     */
+    protected $translationMode = 'default';
+
     /**
      * {@inheritDoc}
      */
     public function init()
     {
+        $this->fillFromConfig(['translationMode']);
+
+        // make the translationMode available to the nestedform formwidgets
+        if (isset($this->config->form)) {
+            $this->config->form['translationMode'] = $this->translationMode;
+        }
+
         parent::init();
         $this->initLocale();
+
+        if ($this->translationMode === 'fields' && $this->model) {
+            $this->model->extend(function () {
+                $this->addDynamicMethod('getJsonAttributeTranslated', function ($key, $locale) {
```

Comment:
Should we prefix this dynamic method with the plugin code? I.e. `winterTranslateGetJsonAttributeTranslated`